proto/process/agent.proto

```diff
@@ -1047,8 +1047,8 @@ message ContainerStatus {
 }
 
 message Manifest {
-	string orchestrator = 1;
-	string type = 2;
+	int32 type = 1;
```

Comment:
Wdyt about moving the type detection from the agent to here as a utility method? 
Similar to what is done to the `TypeCollector<type> detection?
That saves us from duplicating that between our BE and Agent

Reply:
It's a good idea, we can do this once we find a nicer way to mapping the type 👍